src/main/java/org/wildfly/security/auth/client/AuthenticationConfiguration.java

```diff
@@ -630,37 +558,6 @@ public AuthenticationConfiguration useForwardedIdentity(SecurityDomain securityD
         return new SetForwardAuthenticationConfiguration(this, securityDomain, context);
     }
 
-    // Providers
-
-    /**
-     * Use the given security provider supplier to locate security implementations.
-     *
-     * @param providerSupplier the provider supplier
-     * @return the new configuration
-     */
-    public AuthenticationConfiguration useProviders(Supplier<Provider[]> providerSupplier) {
-        return providerSupplier == null ? useDefaultProviders() : new ProvidersAuthenticationConfiguration(this, providerSupplier);
```

Comment:
I'm not sure we should remove this one: providers can be used to locate SASL mechanisms (even if we currently don't) as well as to create/acquire key and password factories (even though we aren't).